app/actions/session.jsx

```diff
@@ -0,0 +1,17 @@
+import Alt from 'alt_flux';
+import sessionSource from 'sources/session';
+
+export default Alt.createActions(class SessionActions {
+  create(user) {
+    return (dispatch) => {
+      sessionSource.create(user).then(result => dispatch(result));
+    };
+  }
+
+  destroy(user) {
+    return (dispatch) => {
+      sessionSource.delete(user);
+      dispatch(user);
+    };
+  }
+});
```

Comment:
just wondering, what do you think about breaking `export default` and class definition into separate lines in order to make construction a bit easier to read?

```js
class SessionActions {
}

export default Alt.createActions(SessionActions)
``` 
?